File: leet-code-solutions-main/1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.js
/**
 * @param {number} n
 * @return {number}
 */
var subtractProductAndSum = function (n) {
    const arr = n.toString().split("");
    let product = 1;
    let sum = 0;
    for (let i = 0; i < arr.length; i++) {
        product = product * parseInt(arr[i]);
        sum += parseInt(arr[i]);
    }
    return product - sum;

};

File: leet-code-solutions-main/1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.java
class Solution {
    public int subtractProductAndSum(int n) {
        int sum=0;
        int multiply=1;
        String num=Integer.toString(n);
        for(int i=0;i<num.length();i++){
            int x=Character.getNumericValue(num.charAt(i)); 
            sum=sum+x;
            multiply=multiply*x;
        }
        return(multiply-sum);
    }
}